devices/generic/items/state_airqualityformaldehyd_item.json

```diff
@@ -0,0 +1,8 @@
+{
+	"schema": "resourceitem1.schema.json",
+	"id": "state/airquality_formaldehyde_density",
+	"datatype": "UInt16",
+	"access": "R",
+	"public": true,
+	"description": "Measured Formaldehyd in microG by m3."
```

Comment:
typo in Formaldehyd, should be Formaldehyde

Reply:
Thx, corrected